pwndbg/disasm/arch.py

```diff
@@ -503,6 +513,26 @@ def _telescope_format_list(self, addresses: List[int], limit: int, emu: Emulator
                 enhance_string_len=enhance_string_len,
             )
 
+    def _enhance_syscall(self, instruction: PwndbgInstruction, emu: Emulator) -> None:
+        if CS_GRP_INT not in instruction.groups:
+            return None
+
+        syscall_register = pwndbg.lib.abi.ABI.syscall().syscall_register
+        syscall_arch = self._get_syscall_arch(instruction)
+
+        instruction.syscall = self._read_register_name(instruction, syscall_register, emu)
+        if instruction.syscall is not None:
+            instruction.syscall_name = (
+                pwndbg.constants.syscall(instruction.syscall, syscall_arch)
```

Comment:
`pwndbg.constants.syscall` takes a `str` for it's second argument, not a `str | None`, which is the type of `syscall_arch`. I'm wondering why `mypy` didn't complain... Anyways, I think it would be more clear to do a `if syscall_arch is None: return None` after line 521 even if it ends up working the way it is.